lib/internal/Magento/Framework/HTTP/Adapter/Curl.php

```diff
@@ -176,8 +176,12 @@ public function write($method, $url, $http_ver = '1.1', $headers = [], $body = '
         } elseif ($method == \Zend_Http_Client::GET) {
             curl_setopt($this->_getResource(), CURLOPT_HTTPGET, true);
             curl_setopt($this->_getResource(), CURLOPT_CUSTOMREQUEST, 'GET');
+        } elseif ($method == \Zend_Http_Client::DELETE) {
+            curl_setopt($this->_getResource(), CURLOPT_CUSTOMREQUEST, "DELETE");
+            curl_setopt($this->_getResource(), CURLOPT_POSTFIELDS, $body);
         }
 
+
```

Comment:
Please remove this line. This will resolve the static test failure.
```
PHP Code Sniffer detected 1 violation(s): 

FILE: ...ar/www/html/lib/internal/Magento/Framework/HTTP/Adapter/Curl.php
----------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 183 | WARNING | Code must not contain multiple empty lines in a row;
     |         | found 2 empty lines.
----------------------------------------------------------------------



Failed asserting that 1 matches expected 0.
```